src/Spreadsheet.tsx

```diff
@@ -157,7 +162,7 @@ class Spreadsheet<CellType, Value> extends PureComponent<{|
       event.preventDefault();
       event.stopPropagation();
       this.clip();
-      this.props.copy();
+      this.props.copy && this.props.copy();
```

Comment:
Ooooh, opportunity to use TS 3.7's optional chaining! https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html
```suggestion
      this.props.copy?.();
```

Reply:
Nice! Very nice. TIL!